ds-sklearn/description.md

```diff
@@ -1,12 +1,19 @@
 This hands-on workshop will provide you with the basics of machine learning using Python.
 
-Machine learning is the field devoted to methods and algorithms that 'learn' from data. 
+Machine learning is the field devoted to methods and algorithms that 'learn' from data.
 It can be applied to a vast range of different domains, from linguistics to physics and from medical imaging to history.
 
-This workshop covers the basics of machine learning in a practical and hands-on manner, 
+This workshop covers the basics of machine learning in a practical and hands-on manner,
 so that upon completion, you will be able to train your first machine learning models and understand what next steps to take to improve them.
 
-We start with processing the data so that it is suitable for machine learning.
-Then we learn how to fit a model to the data using scikit-learn.
-We learn how to select the best model, learn about different machine learning models,
+We start with data exploration so that it is suitable for machine learning.
+Then we learn how to train a model on the data using scikit-learn.
+We learn how to select the best model from the trained models, learn how to
+use different machine learning models (like linear regression, logistic regression,
+decision tree and support vector machine),
 and discuss some of the best practices when starting your own machine learning project.
+
+The workshop will NOT cover the following topics:
+- math and theory on machine learning
+- data proprocessing, e.g. data cleaning, handling missing values
+- deep learning models
```

Comment:
Very good addition!
Should we add references to external courses that cover that? Or is that not our responsibility?

Reply:
No, we don't add the external courses here (it's our advertisement).
We added them to the [resources.md](https://github.com/esciencecenter-digital-skills/workshop-metadata/blob/main/ds-sklearn/resources.md).  We also send out the further study resources to participants after the workshop.